src/forms/Rfc003ParamsForm.tsx

```diff
@@ -59,80 +60,72 @@ function Rfc003ParamsForm({
     return expiry !== "" ? parseInt(expiry, 10) : undefined;
   };
 
-  return (
-    <React.Fragment>
+  function LedgerFields(
```

Comment:
:thinking: I am not too much of a fan of these kind of components where you pass strings and manipulate them to become labels and stuff.

I think the `titleCase` should go away again - are we using this pattern somewhere else still?

Also, you could just make it a proper react component now that it is already a function (you know, with props and stuff).